src/main/resources/com/cloudbees/hudson/plugins/folder/computed/ComputedFolder/orphanedItemStrategyDescription.properties

```diff
@@ -0,0 +1,2 @@
+blurb=Computed folders can remove items immediately or leave them based on a desired retention strategy. \
```

Comment:
It kind of seems like talking about "computed folders" might be even more confusing to users than having no documentation since it is such a general term. I have a similar concern about the term "folder computation".

Maybe better to not have anything here by default and just let subclasses add documentation as desired using the terms that make sense to users in context? I don't feel too strongly either way, so if you think the default here is helpful then it seems fine to me.

Reply:
This is actually the text that is in the official cloudbees documentation :)

Reply:
Ok, if it's already used in official documentation somewhere then it seems best to copy that wording.